### PR TITLE
docs(CONTRIBUTING): fix spelling and link to submit an issue

### PR DESCRIPTION
**Pull request by AdamWendell**
Description: Fixed broken path and spelling error inside CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ that create a lot of child subscriptions, or operators that emit new objects lik
 macro performance tests for.
 
 Other scenarios for macro performance testing may include common end-to-end scenarios from real-world apps. If you have a situation in your
-app where you feal RxJS is performing poorly, please [submit and issue](/ReactiveX/RxJS/issues) and include a minimal code example showing
+app where you feal RxJS is performing poorly, please [submit an issue](https://github.com/ReactiveX/rxjs/issues/) and include a minimal code example showing
```
Comment (kwonoj) [inline CONTRIBUTING.md]: maybe we can update `feal` to `feel` as well?

**Comment by AdamWendell** [comment]
I think I did it correctly. Please tell me if something is incorrect and I will try to fix it.

**Comment by kwonoj** [comment]
I think change's legit. I'll check this in around late today to tomorrow if I can't find anything. Huge appreciate for effort!

**Comment by AdamWendell** [comment]
Found one more typo.

**Comment by kwonoj** [comment]
I came to think it'd be good if there's way to run spell check as similar to run lint over codes per test / CI. :)

**Comment by AdamWendell** [comment]
I  would love too help look for a solution for that.

**Comment by kwonoj** [comment]
Merged with 074b780, appreciate @AdamWendell take effort to bring this. Thanks!

**Pull request merged**